src/tracks/Pattern.cpp

```diff
@@ -55,13 +55,14 @@ QPixmap * PatternView::s_stepBtnOffLight = NULL;
 Pattern::Pattern( InstrumentTrack * _instrument_track ) :
 	TrackContentObject( _instrument_track ),
 	m_instrumentTrack( _instrument_track ),
-	m_patternType( BeatPattern ),
+	m_patternType( MelodyPattern ),
```

Comment:
Just to clarify, what is this change for?

Reply:
This is because by default an empty pattern is a `BeatPattern`, see also `Pattern::checkType()`. When I create an instance of 3osc for example and copy notes from a BB Editor to the Piano Roll of the 3osc, it is assumed to be a beat pattern. The notes are not set to the envelope length then. 
I thought this would be more consistent with the behavior introduced, but after thinking twice I'm not sure about that... What do you think? Is it intended that every instrument gets a BB Pattern on the timeline by default?

Reply:
With `checkType` always making sure the pattern is set to either `BeatPattern` or `MelodyPattern`, it seems like something we don't have to change since it will change itself.

Reply:
Alright, I agree. I'll revert this change.